content/en/llm_observability/quickstart.md

```diff
@@ -54,40 +71,85 @@ To generate an LLM Observability trace, you can run a Python script.
         {"role": "user", "content": "I'd like to buy a chair for my living room."},
     ],
    )
-   {{< /code-block >}}
+   ```
+
+   {{% /tab %}}
+   {{% tab "Node.js" %}}
+
+   ```javascript
+   const { OpenAI } = require('openai');
+
+   const oaiClient = new OpenAI(process.env.OPENAI_API_KEY);
+
+   function main () {
+      const completion = await oaiClient.chat.completions.create({
+         model: 'gpt-3.5-turbo',
+         messages: [
+            { role: 'system', content: 'You are a helpful customer assistant for a furniture store.' },
+            { role: 'user', content: 'I\'d like to buy a chair for my living room.' },
+         ]
+      });
+   }
 
-1. Run the Python script with the following shell command. This sends a trace of the OpenAI call to Datadog.
+   main();
+   ```
+
+   {{% /tab %}}
+   {{< /tabs >}}
+
+3. Run the script with the following shell command. This sends a trace of the OpenAI call to Datadog.
+
+   {{< tabs >}}
```

Comment:
adding tabs here seems to make the page jump when clicking between tabs, but just for this section.

Reply:
I'm going to restructure so that the tabs jump less